File: Java Advanced/stacksAndQueues/SimpleCalculator.java
package stacksAndQueues;

import java.util.*;

public class SimpleCalculator {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String[] tokens = scanner.nextLine().split("\\s+");
        Deque<String> stack = new ArrayDeque<>();
        Collections.addAll(stack, tokens);
        while (stack.size() > 1) {
            int firstNumber = Integer.valueOf(stack.pop());
            String operator = stack.pop();
            int secondNumber = Integer.valueOf(stack.pop());
            if (operator.equals("+")) {
                stack.push(String.valueOf(firstNumber + secondNumber));
            } else {
                stack.push(String.valueOf(firstNumber - secondNumber));
            }

        }
        System.out.println(stack.pop());


    }
}

File: JS Front-End/JS Functions and Statements - Lab/simpleCalculator.js
function simpleCalculator(firstNumber, secondNumber, operator) {
    switch (operator) {
        case 'multiply':
            return firstNumber * secondNumber;
        case 'divide':
            return firstNumber / secondNumber;
        case 'add':
            return firstNumber + secondNumber;
        case 'subtract':
            return firstNumber - secondNumber;
    }
}

console.log(simpleCalculator(5,
    5,
    'multiply'
));